package.json

```diff
@@ -53,7 +53,8 @@
     "@angular-devkit/schematics": "^12.2.2",
     "@typescript-eslint/experimental-utils": "^4.31.1",
     "eslint-etc": "^4.2.4",
-    "eslint-plugin-rxjs": "^3.3.6"
+    "eslint-plugin-rxjs": "^3.3.6",
+    "semver": "^7.3.5"
   },
   "peerDependencies": {
```

Comment:
Let me know your opinion on upgrade those `peerDependencies` + adding `@ngrx` + `rxjs`..

Reply:
Oh yea, I forgot...
I don't think that's a good idea, this plugin covers 3 separate ngrx packages.
It's perfectly fine to not have one of them installed.
Maybe with optional dependencies like [this](https://github.com/timdeschryver/ngrx-immer/blob/main/src/package.json#L31) ? As long as it isn't needed, I wouldn't add them though.

Reply:
You have a point about `@ngrx/*` stuff... let's not add them so.